lib/plugins/package/lib/zipService.js

```diff
@@ -75,3 +103,71 @@ module.exports = {
     });
   },
 };
+
+function excludeNodeDevDependencies(servicePath) {
+  const cwd = process.cwd();
+  let exclude = [];
+  let include = [];
+
+  try {
+    const packageJsonFilePaths = globby.sync([
+      '**/package.json',
+      // TODO add glob for node_modules filtering
+    ], {
+      cwd: servicePath,
+      dot: true,
+      silent: true,
+      follow: true,
+      nosort: true,
+    });
+
+    // filter out non node_modules file paths
+    const relevantFilePaths = _.filter(packageJsonFilePaths, (filePath) => {
+      const isNodeModulesDir = !!filePath.match(/node_modules/);
+      return !isNodeModulesDir;
+    });
+
+    _.forEach(relevantFilePaths, (relevantFilePath) => {
+      // the path where the package.json file lives
+      const fullPath = path.join(servicePath, relevantFilePath);
+      const rootDirPath = fullPath.replace(path.join(path.sep, 'package.json'), '');
+
+      process.chdir(rootDirPath);
+
+      // TODO replace with package-manager independent directory traversal?!
+      const prodDependencies = childProcess
+        .execSync('npm ls --prod=true --parseable=true --silent')
+        .toString().trim();
+
+      const prodDependencyPaths = prodDependencies.match(/(node_modules\/.*)/g);
```

Comment:
@pmuens This does not work with Windows. `\/` has to be replaced with `path.sep`.

The output of `npm ls --prod=true --parseable=true --silent` under Windows returns backslashes as path separators.